From 2b5823b2053414f8c56c9d5a1e5d452f0366790a Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Tue, 14 Apr 2015 16:10:09 +0800
Subject: [PATCH] MLK-10653 ARM: dts: imx7d-sdb: disable hdmi due to pin
 conflict with epdc

EPDC_DATA13 is used by HDMI and EPDC, disable HDMI for EPDC.

Signed-off-by: Robby Cai <r63905@freescale.com>
(cherry picked from commit 6d932a92b03c44f17bf52167aeefc3e13aeb932c)
---
 arch/arm/boot/dts/imx7d-sdb-epdc.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dts b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
index 5721824..983b4f8 100644
--- a/arch/arm/boot/dts/imx7d-sdb-epdc.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
@@ -23,3 +23,7 @@
 &max17135 {
         status = "okay";
 };
+
+&sii902x {
+	status = "disabled";
+};
-- 
1.8.0

